tests/benchmarks/micro_benchmarks/test_cublaslt_function.py

```diff
@@ -37,26 +38,28 @@ def test_cublaslt_gemm_cls(self):
 
     def test_mrange(self):
```

Comment:
Why do we need to change this?

Reply:
since I moved this function out of the class so other benchmark can also reuse this function